### DISCUSSION TITLE
Dudas sobre las practicas de introduccion.

### DISCUSSION THREAD

**Discussion by OscarIslasH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#94] [Q&A] [answered]
Ayuda, no se como hacer las practicas o lo que debemos hacer, vi el primer y segundo stream pero el segundo no me deja abrirlo, en el tercero ya no entendi nada. Apenas me uni el 20 por falta de tiempo y quisiera saber que es lo que tengo que hacer, mas que nada por que primero entendi que no tenia que ver con codigo y luego que si, la verdad he estado muy ocupado y hasta ahora puedo empezar, si alguien pudiera explicarme, se los agraeceria mucho :(

En el primero entendi: Que solo hay que rellenar el primer documento, pero no se en base a que o si es como yo quiera enteramente.

 En el segundo segun lo que entendi habia que editar el pdf a forma personal con mis datos pero realmente no se.

Por favor si alguien me puede explicar lo que tengo que hacer a grandes rasgos se los agradeceria bastante por que en serio estoy perdido.

**Comment by Burstniart** [comment]
Las practicas son sobre la toma de requerimientos para un proyecto de software, la comunicacion con el cliente, definicion de publico objetivo y buyer persona, y el design
Investiga un poco sobre la toma de requerimientos de software, hay varios formatos para llenar un documento
La parte de design la puedes ir revisando en el temario, desde hacer un boceto de la aplicacion, diferentes diagramas que se usan para ir definiendo la aplicacion, su flujo, el flujo de datos, como sera el layout y el design

**Reply by OscarIslasH** [reply]
Entiendo el fin, pero lo que no entendi es como debo rellenar el formato, por ejemplo, entiendo que el proyecto es "llenado de formulario" Pero que pongo en "nombre de requerimiento" "fecha de solicitud" 

Es lo que me tiene confundido y atorado :c 
Se el fin de la practica pero no se como hacerla y cuando vine a ver las discussions solo decian el fin pero no como o en que basarnos para resolverlas, busque en internet sobre la toma de requerimientos para un proyectos pero no se con que rellenar el documento.

**Reply by patricia2454** [reply]
Busca videos de apoyo, si se te complica el formato, tu puedes hacer el propio, es el camino que se sigue, para identificar los alcances del proyecto, desde el momento de la entrevista inicial, hasta la propuesta e implementación de la página web; busca videos así tal cual: ¿cómo se llena un formato de requeremientos de software? y así con lo que sigue: ¿cómo se realiza un frimeware? ¿cómo se hace un buyer persona?  etc

**Comment by GloriaCoralCerecedo** [comment, marked as answer]
Hola, mira tranquilo no te estreses. El MC dijo que las plantillas que dio o las recomendaciones son opcionales, puedes usar otras más básicas o que tu endientas. 

1. En el de requerimientos, llena lo que tu comprendas, puedes quitar poner, colocar 1, 2, 3, 5, 6 requerimientos es dependiendo tu análisis y entendimiento. Y si te atoras mucho, vete al  siguiente punto. Igual yo estoy batallando un poco avance pero no he terminado aun 
2. El buyer person yo lo hice con una plantilla https://www.hubspot.es/make-my-persona, ahí más o menos igual te explica
3.  El público objetivo de qui lo estoy haciendo, esto lo recomendó el MC https://miro.com/es/plantillas/publico-objetivo/, igual aqui te explica 
4. Lo demás puntos no lo he hecho pero dio herramientas y creo hay tutoriales

Pero tranquilo, recuerda no hay respuestas incorrectas.

Además tengo entendido que es para el 25, pero esa es una fecha ideal para que no te atrases. Pero creo que se puede enviar un poco más adelante como el 28 por ejemplo.

**Reply by Mavoportentous** [reply]
¿Todo lo que vaya haciendo lo subo a un repositorio? y muchas gracias por la información

**Reply by GloriaCoralCerecedo** [reply]
Si, el MC dijo que puede ser un repositorio por semana o en un solo repositorio vayas subiendo todo. Es como quieras.

Igual cualquier duda puedes preguntarle.  O ver las discusiones de aquí. 

Igual el 25 dirá cómo subirlo, es como las latas de python, nos va a dar un formulario y ahí colocamos el link del repo

**Reply by SantQuintana** [reply]
@OscarIslasH Veo que en la pregunta pones que si llenas tus datos y pues no, ahi va la informacion de tu buyer persona, es como crear un "avatar" de la persona ideal que compra/usaría tu producto, cuales son sus gustos, edad hobbies, hastacuanto dinero gana y esas cosas.

**Reply by OscarIslasH** [reply]
Acabo de hacer el buyer pero aun se me dificulta un poco la primera, en donde dice "logo" debo poner un logo de los abogados? es que en serio no se si deba rellnarlo todo o lo maximo posible

O si el primero es en base al buyer

**Reply by RodolfoMorquecho** [reply]
Hola Gloria, en el primer punto donde van los requerimientos tu usaste un caso puntual ?(por ejemplo que el abogabot este enfocado en personas que realicen demandas de pensión alimenticia unicamente) o lo dejaste de manera general y que acepte cualquier demanda?
Te pregunto esto porque a la hora de realizar el buyer persona es complicado si dejo generalizado el primer punto, no?

**Reply by GloriaCoralCerecedo** [reply]
Hola, pues fíjate que primero pensaba con demandas civiles que son más comunes pero luego me fui por lo general, (civil, penal, mercantil). Si es algo complicado. En el buyer person en mi caso hice 3, uno del abogado, otro de un trabajador independiente y otro de una empresa. Y el público objetivo bueno es más general.

Igual lo que pienso es que cualquiera que se el tipo de demanda, el proceso será similar, ya que cuando mande el formulario con el administrador se Genera el formato de Word ahí me imagino estará acomodado como debe ir, por eso el formulario debe ser específico. 

Pero lo puedes hacer como piensas tú, cada quien tiene su tipo de análisis y punto de vista

**Reply by RodolfoMorquecho** [reply]
Gracias por responder, concuerdo el proceso de requerimientos sera similar en varios casos de demandas pero en el buyer si existen diferencias. Pero me parece buena tu idea de generar un buyer para cada caso.

**Reply by GloriaCoralCerecedo** [reply]
Si así es, aunque el MC recomendó que 1 era ideal. Pero en la plantilla te dice que puedes 3 a 5. Igual en caso de ser uno pudieras elegir al que consideras más ideal.

**Reply by OscarIslasH** [reply]
@GloriaCoralCerecedo El publico objetivo son los abogados del caso de abogabot o los clientes de los abogados? ahi me perdi un poco, por que si bien el sofware seria vendido y distribuido para que los abogados trabajen, los que acabarian usandolo serian los clientes 

Es lo unico que no me ha quedado del todo claro, hice el buyer persona en base a los abogados, asi que si me pudieras aclarar eso, te lo agradeceria bastante.

**Reply by AldoSpezzia** [reply]
Muchas gracias me ayudo bastante!!!!!

**Comment by robertogarcer** [comment]
comparto este video donde se explica un poco mas el primer documento 

https://www.youtube.com/watch?v=2KZKMY75cJM

**Comment by AndresAVargas** [comment]
Para el publico objetivo lo explico, vi los lives pero no me quedo muy claro que llenar

**Reply by OscarIslasH** [reply]
Publico objetivo se refiere a: A que publico quieres llegar, en el buyer persona que puedes hacer en la pagina: https://www.hubspot.es/make-my-persona vas rellenando los datos que te pide conforme al caso del abogado, en este caso tendras que buscar en internet todo lo que viene ahi, por ejemplo "Responsabilidades laborales" debes poner cuales son las resmponsabilidades laborales de los abogados

Al igual el primero, tiene que ser en base al tema de los abogados, en nase a un cliente que sea un abogado, vas viendo cada casilla e investigas que significa y como se acopla a los abogados, hay que rellenar y hacer todo en base al caso de abogabot que quiere una plataforma, en este caso, el publico objetivo tengo entendido son los abogados o clientes de los abogados, aun no me ha quedado del todo claro pero es lo que entendi

**Reply by OscarIslasH** [reply]
Ya vi todo lo demas y es del demandante o cliente el publico objetivo, tambien volvi a ver los lives y justo explican que si bien el comprador puede ser una persona, el publico objetivo es al que llega.

Pusieron el ejemplo de una compra que hizo un padre para su hijo, si bien el padre compro un producto, el consumidor sera el hijo, el producto tiene como publico objetivo al niño.

Igualmente abajo de el link que mande explica mejor que es un publico objetivo, te lo pongo aqui por si acaso:

Sobre la plantilla de público objetivo
¿Qué es un público objetivo?
Tu público objetivo es tu grupo de clientes potenciales. Es el conjunto de personas que podría estar interesada en tu producto o servicio. Encontrar tu público objetivo es un paso fundamental para el marketing eficaz, el desarrollo de productos y la estrategia de negocios general.

¿Qué es el análisis de público objetivo?
Los análisis de público objetivo consisten en describir a tu audiencia en términos de una variedad de parámetros demográficos, como la edad y el género, y de otras variables como ingresos, educación y ubicación o psicográficas como intereses y opiniones. Analizar el público objetivo te ofrece información valiosa para la mayoría de las funciones de negocios.

Cómo usar la plantilla de público objetivo
Según tus objetivos, puedes usar la plantilla de público objetivo de muchas formas. Para algunas campañas de marketing, quizás te interesen los datos demográficos como edad y género. Para otras campañas, o para la creación de productos, podrían interesarte variables psicográficas como intereses y opiniones.

La plantilla es especialmente útil si estás organizando varias campañas de marketing o de anuncios al mismo tiempo, o si estás comercializando un producto mientras tratas de lanzar otro. Te permite estar organizado, mantener el dinamismo y a tus equipos alineados. Estos son algunos elementos clave que podrías incluir en una plantilla de este tipo.

Paso 1: Recopilar datos demográficos

Los datos demográficos son un conjunto de criterios que puedes usar para describir un segmento de la población. Algunos datos demográficos incluyen: edad, género, ingresos, nivel educativo, estado civil y ocupación. Facebook Audience Insights y Google Trends pueden ayudarte a diseccionar los datos demográficos que luego incluirás en la plantilla.

Paso 2: Recopilar datos de ubicación

Incluso si tu empresa es internacional, estos datos pueden ayudarte a reducir el público objetivo por criterios geográficos o de ubicación. Por ejemplo, quizás tus clientes se concentran en un país o región determinado. Al definir tu público por ubicación, puedes corroborar que te diriges a esas áreas específicas. Localiza tu público objetivo por barrio, ciudad, estado, país o región. Para esto también podrían resultar útiles Facebook Audience Insights y Google Trends, al igual que MyBestSegments.

Paso 3: Recopilar datos psicográficos

La información psicográfica incluye actitudes, opiniones, intereses y actividades. Por ejemplo, es posible que te interese dirigirte a aficionados a los videojuegos, escaladores, coleccionistas de juguetes o amantes de la comida. Estos parámetros pueden ser un poco más difíciles de medir que los demográficos y los datos de ubicación. Para extraerlos, podrías necesitar realizar una encuesta. Si las encuestas no son una posibilidad, puedes usar Facebook Audience Insights para filtrar públicos por intereses u opiniones.

Los datos psicográficos son potentes porque pueden ser muy granulares. En lugar de filtrar solo por aficionados a los videojuegos, puedes dirigirte a audiencias que prefieren juegos de mesa desarrollados por creadores independientes; en lugar de dirigirte únicamente a coleccionistas de juguetes, puedes buscar coleccionistas que disfrutan de salir a buscar ofertas. Estos datos te permiten profundizar en los perfiles de cliente ideal específicos que están interesados en tu producto o servicio.

Paso 4: Selecciona de dos a cinco identificadores

Ahora que tienes todos estos datos, es momento de reducirlos. ¿Por qué elegir entre 2 y 5 si tienes toda esa información con la que trabajar? Si eliges solo uno o dos, tu público será tan masivo que sería lo mismo si te dirigieras al público en general. Si los seleccionas a todos, tu público será tan pequeño que no valdrá la pena ponerse a trabajar en él.

Paso 5: Pon tu público a prueba

En este punto del proceso, tienes una mejor idea sobre cuál es tu público objetivo. Para averiguar si estás en lo correcto, puedes poner tu público a prueba. Aprovecha los contactos directos con las personas que podrían entrar en tu mercado objetivo. Si estás comercializando una aplicación de citas para entusiastas del ciclismo, por ejemplo, podrías ir a un campus universitario y mostrarles a los estudiantes prototipos de la aplicación. Si lo haces, registra sus opiniones. Pregúntales si han usado un servicio similar y si tienen alguna crítica. Si nunca usaron algo similar, pregúntales si estarían dispuestos a hacerlo en el futuro. Si el público objetivo parece entusiasmado, sabes que estás en el camino correcto.

Si es difícil acercarte a tu público objetivo en persona, puedes recurrir a herramientas online para lograr los mismos resultados. Los subreddits, grupos de Facebook y foros son una buena fuente de datos.

Cuando hayas evaluado a tu audiencia, no tengas miedo de volver a la plantilla. Haz ajustes, divide los datos de otra forma y realiza experimentos para acercarte más a tu objetivo.

4. Ejemplos de público objetivo
Para entender mejor a los públicos objetivo, ten en cuenta los siguientes ejemplos:

1. El público objetivo de Men's Warehouse

El público objetivo de Men's Warehouse no está conformado solo por hombres que necesitan atuendos formales, sino también por las esposas y novias que pueden sugerir o influenciar dónde compran sus compañeros. Al comprender al público objetivo en sentido amplio, pueden comercializar con todos los clientes posibles.

2. El público objetivo de una campaña de marketing de McDonald's.

Una empresa puede tener públicos objetivo diferentes para cada campaña de marketing. Por ejemplo, McDonald’s podría tener dos campañas de marketing diferentes para su menú de desayuno. Una que apunte a trabajadores de oficina que quieran ahorrar tiempo en el desayuno antes de entrar a trabajar y otra que apunte a trabajadores que busquen una comida caliente para llevar al lugar de trabajo.

3. El público objetivo de un bufete jurídico

Un bufete jurídico especializado en daños personales tendrá un gran público objetivo entre quienes hayan resultado lesionados o entre quienes trabajen en sectores donde las lesiones son comunes. Su objetivo sería asegurarse de que quienes tengan mayor posibilidad de necesitarlos los conozcan.

4. El público objetivo de un nuevo negocio o startup

Las nuevas empresas, especialmente las que todavía no lanzaron su producto o servicio, suelen tener dificultades para encontrar su público objetivo. De hecho, el público objetivo de una empresa nueva puede cambiar con el tiempo a medida que la empresa crece y recibe comentarios sobre el producto.

¿Por qué deberías encontrar tu público objetivo?
Encontrar el público objetivo es crucial. Cada producto y campaña de marketing es una inversión significativa de tiempo y recursos. Para asegurarte de que estás usando esos recursos de manera eficaz, es importante que sepas a quién diriges tu campaña de marketing. Encontrar tu público objetivo te ayuda a despachar grandes productos y crear campañas de marketing que atraigan multitudes.

¿Por qué es importante el público objetivo?
Los clientes son el alma de todas las empresas. La corporación más gigantesca y la startup más desconocida tienen un objetivo en común: conseguir más clientes. Para lograrlo, sin embargo, tienen que saber quiénes son. Por eso es crucial descubrir tu público objetivo.

Estudios recientes sugieren que más de un tercio de las empresas tienen problemas para diseñar campañas de marketing eficaces, y la mitad de ellas no saben como llegar a clientes potenciales. Encontrar tu público objetivo es el primer paso para resolver estos problemas. Los datos del público objetivo dan forma a tus productos y servicios, precios, palabras clave de marketing, elecciones publicitarias y diseño.

Cuándo usar la plantilla de público objetivo
Usa la plantilla de público objetivo cuando quieras entender quién está interesado en tus productos o servicios. Puede que esto te resulte especialmente beneficioso cuando estés poniendo en marcha varias campañas de marketing o de anuncios o haciendo malabares con varios proyectos al mismo tiempo.

Preguntas frecuentes sobre la plantilla de público objetivo
¿Por qué es importante el púbico objetivo?
Los clientes son el alma de todas las empresas. La empresa más grande y la startup más pequeña tienen un objetivo en común: conseguir más clientes. Sin embargo, para hacerlo, es importante saber quiénes son. Por eso es fundamental descubrir a su público objetivo. Estudios recientes sugieren que más de un tercio de las empresas tienen dificultades para diseñar campañas de marketing efectivas y la mitad no sabe cómo llegar a clientes potenciales. Encontrar el público objetivo es el primer paso para resolver estos problemas. Los datos de la audiencia objetivo informan sus productos y servicios, precios, palabras clave de marketing, opciones publicitarias y diseño.

¿Por qué debes buscar tu público objetivo?
Encontrar tu público objetivo es fundamental. Cada producto y campaña de marketing es una inversión significativa en tiempo y recursos. Para asegurarte de que estás utilizando esos recursos de manera eficaz, es importante saber a quién te diriges. Encontrar tu público objetivo te ayuda a enviar excelentes productos y a crear campañas de marketing que agraden al público.

¿Cómo hacer un escrito correctamente?
Una declaración de audiencia objetivo debe ser concisa y debe de comunicar la información previamente descubierta en la investigación sobre los consumidores. El género, la edad y los intereses son obligatorios en un buen ejemplo de público objetivo. También puedes agregar a su declaración un punto de vista, que le da más contexto sobre cómo hiciste la investigación y cómo se te ocurrió esta información.

**Comment by AlexZRamirez** [comment]
Para quien le quedará duda acerca del publico objetivo, hay una guía de facebook que puede ayudarles a comprender un poco mejor el tema. https://www.facebook.com/business/learn/lessons/create-target-audience

**Reply by OscarIslasH** [reply]
Pero el publico objetivo si es el abogabot o el cliente del abogabot?

**Reply by romarpla** [reply]
https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/131 Aquí está tu respuesta!
